### PR TITLE
guidelines: add leaving organization check-list

### PR DESCRIPTION
**Pull request by 0xc0170**
Any person leaving our organization should follow these guidelines.

@AnotherButler Please review - I added new section to adding yourself to the organization - leaving it.

### CODE REVIEW

**Comment by 0xc0170** [comment]
@ARMmbed/mbed-os-maintainers

File: docs/reference/contributing/guidelines/guidelines.md [hunk 1]
```diff
@@ -46,6 +46,10 @@ If so, you can request to become an organization member, but you must first ensu
 - Arm staff must have their Name, Company (Arm) and Location publicly visible.
 - All others should have their Name and Company visible. Including your Location helps us interpret response times according to time zones.
 
+### Leaving ARMmbed organization
+
+Best practices for leaving the organization are documented on [Github help](https://help.github.com/en/articles/best-practices-for-leaving-your-company). Please follow these best practices once you leave the organization.
+
```
Comment (adbridge) [inline docs/reference/contributing/guidelines/guidelines.md:52]: I would rewrite this to say something like:

When leaving the company please follow the following best practice: <link>

**Review by adbridge — CHANGES_REQUESTED** [review]

File: docs/reference/contributing/guidelines/guidelines.md (hunk 1 above)
Comment (0xc0170) [reply docs/reference/contributing/guidelines/guidelines.md:52]: fixed

**Review by AnotherButler — APPROVED** [review]
Thanks for the PR 👍

**Review by adbridge — APPROVED** [review]

**Comment by 0xc0170** [comment]
Ready!

**Pull request merged**